### DISCUSSION TITLE
https://plume.pengzhanbo.cn/config/navigation/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#67] [docs-comment]
# https://plume.pengzhanbo.cn/config/navigation/

概述 主题默认会自动生成最简单的导航栏配置，仅包括 首页 和 博客文章列表页 。 你也可以自己配置导航栏，覆盖默认的的导航栏配置。 默认配置如下： 当开启了 多语言配置，则会生成对应语言的导航栏的默认配置： 但是，通常情况下，随着站点内容变得越来越丰富，包括了 博客文章列表、notes、友情链接、 外部链接等等， 默认生成的导航栏配置满足不了您的需求。...

https://plume.pengzhanbo.cn/config/navigation/

**Comment by MemoryZy** [comment]
效率高👍🏻
